util.py

```diff
@@ -173,17 +192,47 @@ def kill_robot(self, robot):
 
 class BeltTile(Tile):
     def __init__(self, direction, path, properties):
-        self.crossroads = properties[0]["value"]
-        self.belt_direction = properties[1]["value"]
-        self.move_count = properties[2]["value"]
+        self.direction_out = self.transform_direction(properties[0]["value"])
+        self.express = properties[1]["value"]
         super().__init__(direction, path, properties)
 
-    def move_robot(self, state):
-        # TO DO!
-
-        # 1) Express belts move 1 space
-        # 2) Express belts and normal belts move 1 space
-        pass
+    def transform_direction(self, direction_integer):
```

Comment:
Nelíbí se mi úplně nápad psát další metodu na stejnou věc. Již máme v utilu funkci `transform_direction`, která by se dala rozšířit, aby pokryla i tyto transformace. Můžeme pro zpřehlednění změnit properties map, aby se stringy "left", "right" u `GearTile` načítaly jako integery `-90`, `90`

Reply:
Ano to máš pravdu, taky se mi to úplně nelíbí a mám v plánu to nějak sjednotit. Je to pozůstatek z doby, kdy to ještě nebyla samostatná funkce.